src/pages/billing/List.ts

```diff
@@ -0,0 +1,37 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { ListPage } from 'src/billing-ui/List/ListPage';
+import FooterWrapper from 'src/components/FooterWrapper';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+
+interface ListProps {
```

Comment:
The remainder of https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/billing/List/List.ts